v2/brigadier/src/jobs.ts

```diff
@@ -20,33 +61,177 @@ export class Job {
     this.event = event
   }
 
+  /**
+   * Runs the job.
+   * 
+   * When you run the job, Brigade runs all the containers, primary and sidecar.
+   * When the primary container exits, the job ends. If sidecars are still running
+   * at this point, Brigade terminates them after a short delay.
```

Comment:
This "short delay" language is perfect... might be the answer to resolving this [other comment](https://github.com/brigadecore/brigade/pull/1292/files?file-filters%5B%5D=.ts#r591631849)